deploy/stash.sh

```diff
@@ -13,9 +13,61 @@ kubectl config current-context || {
 }
 echo ""
 
+OS=""
+ARCH=""
+DOWNLOAD_URL=""
+DOWNLOAD_DIR=""
+TEMP_DIRS=()
+ONESSL=""
+HELM=""
+
 # http://redsymbol.net/articles/bash-exit-traps/
 function cleanup() {
```

Comment:
What if we can just add this as the installer script for the `stashed/postgres` repo and complicate the installer script for the main repo?

Reply:
removed